common-arcana.lic

```diff
@@ -344,7 +344,7 @@ module DRCA
     aura
   end
 
-  def cast_spells(spells, settings, force_cambrinth = false)
+  def cast_spells(spells, settings, force_cambrinth = false, location = nil)
```

Comment:
❓ This will use the location for every spell in `spells`. Is that desirable? Location feels to me like it would/should be specific to a single spell.

Reply:
Not sure how many spells need something other than "cast".
This could be used for any of them. "cast player", "cast left arm internal reverse", etc

Reply:
https://elanthipedia.play.net/Cast_command
There are quite a few different casting options.

Reply:
This function is used to cast multiple spells. As written, this will use location for all of them.

Is that intended?